spec/requests/api/v2/providers/courses/update_spec.rb

```diff
@@ -266,8 +266,8 @@ def perform_request(course)
         {
           about_course: "",
           fee_details: "",
-          fee_international: "",
-          fee_uk_eu: "",
+          fee_international: 0,
```

Comment:
This looks wrong to me. 0 has a different meaning to empty string.

Reply:
Correct, but this is the same implementation as the c# app.